docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Accessing the Cloud with Viam's Client SDKs"
+linkTitle: "Cloud Management"
+weight: 20
+type: "docs"
+description: "How to use the cloud API with Viam's client SDKs."
+tags: ["cloud", "sdk", "viam-server", "networking", "apis", "robot api", "cloud management"]
+---
+
+The cloud app API allows you to [manage your robot fleet](/manage/fleet/) with code instead of through the [Viam app](https://app.viam.com/).
```

Comment:
I think this is slightly misleading because if I understand correctly it is still through the app api regardless, we're just abstracting it to an interface or not. You could say something only slightly different: 
```suggestion
Use the cloud app API [manage your robot fleet](/manage/fleet/) with code instead of in the [Viam app](https://app.viam.com/).
```

Reply:
Mmm good point; this could be made more clear. How about this:
```suggestion
The cloud app API allows you to [manage your robot fleet](/manage/fleet/) with code instead of with the graphical interface of the [Viam app](https://app.viam.com/).
```

Reply:
I don't think we reaslly need to spell out GUI but that look good!

Reply:
I initially didn't think we did but I think this makes it more explicit what we mean? Since "in" vs "through" isn't that explicit